apps/web/src/components/ui/EditQuestionBankButton.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger
+} from "@/components/ui/dialog";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import { Label } from "@/components/ui/label";
+import { Textarea } from "@/components/ui/textarea";
+import { RadioGroup, RadioGroupItem } from "@/components/ui/radio-group";
+interface Props {
+  // initialData: any[];
+  initialData: unknown;
+}
+export const EditQuestionBankButton = ({ initialData }: Props) => {
+  // const { data: questionBanksData } = trpc.questionBank.list.useQuery(
+  //   undefined,
+  //   {
+  //     initialData
+  //   }
+  // );
+  const questionBanksData = initialData;
```

Comment:
remove comments

Reply:
Resolved in https://github.com/peiyuans-pals/toohakai/commit/f14910ed45d4e3bd13b9665ed2175dab6986f459